### ISSUE TITLE
Core dump with 4.4 branch @ cf3457f

### ISSUE THREAD

**Issue by amessina** [kamailio/kamailio#560]
When attempting to start kamailio built from the 4.4 branch at commit cf3457f, it dumps core.
```
Reading symbols from /usr/sbin/kamailio...Reading symbols from /usr/lib/debug/usr/sbin/kamailio.debug...done.
done.
[New LWP 19570]
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib64/libthread_db.so.1".
Core was generated by `/usr/sbin/kamailio -c'.
Program terminated with signal SIGSEGV, Segmentation fault.
#0  0x0000000000447026 in cnt_hash_add (group=0x71fd26, name=0x7202b8, flags=369, cbk=0x71fd10 <__func__.6293>, param=0x6371d5 <fm_realloc+4612>, doc=0x7ffd0c70dd30 "\f\336p\f\375\177") at counters.c:328
328             doc_len = doc?strlen(doc):0;
(gdb) bt full
#0  0x0000000000447026 in cnt_hash_add (group=0x71fd26, name=0x7202b8, flags=369, cbk=0x71fd10 <__func__.6293>, param=0x6371d5 <fm_realloc+4612>, doc=0x7ffd0c70dd30 "\f\336p\f\375\177") at counters.c:328
        e = 0x0
        cnt_rec = 0x489159e
        grp_rec = 0x7ffd0c70e2a0
        p = 0x0
        v = 0xffffffffb8ec5010
        doc_len = 0
        n = 32691
        __func__ = "cnt_hash_add"
#1  0x00000000004491b8 in counter_get_name (handle=...) at counters.c:684
        __llevel = 32691
        __func__ = "counter_get_name"
#2  0x000000000044b6eb in ip_addr2sbuf (ip=0x771d60, buff=0xa081e0 <core_cfg_def+544> "\200ܫ", len=0) at ./ip_addr.h:658
        __llevel = 1
        __func__ = "ip_addr2sbuf"
#3  0x000000000051aaa0 in fix_sock_str (si=0xffffffff0c70e1e0) at socket_info.c:443
        __llevel = 0
        len = -1
        __func__ = "fix_sock_str"
#4  0x00007fb3b96e6580 in __libc_start_main (main=0x517fd2 <add_alias+381>, argc=2, argv=0x7ffd0c70e2a8, init=<optimized out>, fini=<optimized out>, rtld_fini=<optimized out>, stack_end=0x7ffd0c70e298) at libc-start.c:289
        result = <optimized out>
        unwind_buf = {cancel_jmp_buf = {{jmp_buf = {0, -7223469352753121223, 4288480, 140724812178080, 0, 0, 7222380612212668473, 7252597343634765881}, mask_was_saved = 0}}, priv = {pad = {0x0, 0x0, 0x712580 <__libc_csu_init+32>, 
              0x7ffd0c70e2a8}, data = {prev = 0x0, cleanup = 0x0, canceltype = 7415168}}}
        not_first_call = <optimized out>
#5  0x0000000000417009 in _start ()
No symbol table info available.
```

**Comment by amessina** [comment]
Reverting 4992dfbb9f15cfb8ade99be4e6d3087146cb8e22 and e6dd2c172be3712e8c468cbe7e55ba94d81a7b62 prevent the core dump.

**Comment by miconda** [comment]
Can you try with latest branch 4.4 or master, there were fixes related to this.

**Comment by amessina** [comment]
@miconda, I was just going to say... this is resolved with 6684be6b4c405ab24dd6a5a19267ceef6aa95503 and 49ec5fa5d9bb939b80ef3994b5f0c04e7525ab9f. Thank you!
